index.html

```diff
@@ -64,75 +61,107 @@ <h1 class="carousel-heading">Join role playing campaings</h1>
         <span class="sr-only">Next</span>
       </a>
     </div>
-    </header>
-    <nav class="navig container-fluid">
-        <ul class="lista row ">
-            <li
-                class="lista_home col-md-3">
-                <a
-                    href="#hero">HOME</a>
-            </li>
-            <li
-                class="lista_about col-md-3">
-                <a
-                    href="#function">ABOUT</a>
-            </li>
-            <li
-                class="lista_authors col-md-3">
-                <a
-                    href="#authors">AUTHORS</a>
-            </li>
-            <li
-                class="lista_extras col-md-3">
-                <a
-                    href="#extras">EXTRAS</a>
-            </li>
-        </ul>
-    </nav>
+  </header>
+  <nav class="navig container-fluid">
+    <ul class="lista row ">
+      <li class="lista_home col-md-3">
+        <a href="#hero">HOME</a>
+      </li>
+      <li class="lista_about col-md-3">
+        <a href="#function">ABOUT</a>
+      </li>
+      <li class="lista_authors col-md-3">
+        <a href="#authors">AUTHORS</a>
+      </li>
+      <li class="lista_extras col-md-3">
+        <a href="#extras">EXTRAS</a>
+      </li>
+    </ul>
+  </nav>
 
-    <section class="container-block section-authors">
-      <div class="section-authors-title">
-        <div>
-            <h1 class="carousel-heading">AUTHORS</h1>
-            <p class="carousel-paragraph">The men who play</p>
-        </div>
+  <section class="functions--content">
+    <h1 class="functions--content--header">Functions</h1>
+    <div class="functions--content--container">
+      <div>
+        <h2 class="functions--content--header">1. Choose game</h2>
+        <img src="./Ikony/kufer2.png" class="functions--img" alt="Game">
+        <p class="functions--content--paragraph">
+          Use the list of most popular games, or add one by your own.
+        </p>
+      </div>
+
+      <div>
+        <h2 class="functions--content--header">2. Gather players</h2>
+        <img src="./Ikony/flgi.png" class="functions--img" alt="Players">
+        <p class="functions--content--paragraph">
+          Decide how many players you are looking for.
+        </p>
+      </div>
+
+      <div>
+        <h2 class="functions--content--header">3. Fix place and date</h2>
+        <img src="./Ikony/zwój3.png" class="functions--img" alt="Place and date">
+        <p class="functions--content--paragraph">
+          Select location, date and time of your desired game session.
+        </p>
+      </div>
+
+      <div>
+        <h2 class="functions--content--header">4. Reserve table</h2>
+        <img src="./Ikony/piwo.png" class="functions--img" alt="Reserve table">
+        <p class="functions--content--paragraph">
+          Select a venue and reserve a table.
+        </p> 
       </div> 
-      <div class="information-authors">
```

Comment:
why authors sections is removed?

Reply:
Cleaning sections and divs. It's not deleted, but moved out of Functions section and secured within it's own Authors section down the code (lines 93-98 and 100-120 marked red are in lines 118-123 and 125-152).